newa/__init__.py

```diff
@@ -106,6 +108,52 @@ def render_template(
         raise Exception("Could not render template.") from exc
 
 
+@define
+class Settings:
+    """ Class storing newa settings """
+
+    et_url: Optional[str] = None
+    rp_url: Optional[str] = None
+    rp_token: Optional[str] = None
+    rp_project: Optional[str] = None
+    jira_url: Optional[str] = None
+    jira_token: Optional[str] = None
+    jira_project: Optional[str] = None
+    tf_token: Optional[str] = None
+    tf_recheck_delay: Optional[str] = None
+
+    def get(self, key: str, default: Optional[str] = '') -> str:
```

Comment:
Can the `default` ever be `None`? Even the default value is a string. If it can be `None`, then there is a surprise waiting in the code below: `settings.get('foo', default=None) -> getattr(self, 'foo', None) -> str(None) -> get() returns "None"`. I'd recommend dropping either `Optional`, or `str`, either one should go: `Optional` does not seem to apply in any call, and if it does and `None` should be allowed, `str()` will mask invalid keys, providing bogus return values for them.

Reply:
I must admit that my understanding when `Optional` should be used was wrong. I have dropped it from the code on few other places and well.